src/main/java/com/google/devtools/build/lib/analysis/test/TestConfiguration.java

```diff
@@ -84,6 +86,24 @@ public static class TestOptions extends FragmentOptions {
                 + "-1 tells blaze to use its default timeouts for that category.")
     public Map<TestTimeout, Duration> testTimeout;
 
+    @Option(
+        name = "test_resources",
```

Comment:
Nit: rename to  `--default_test_resources`. I would like to distinguish this from `--local_resources` in that the former is about the resource usage of each test action, and the latter is about the resources available to Bazel.